packages/desktop-client/src/components/FinancesApp.js

```diff
@@ -270,6 +272,14 @@ class FinancesApp extends React.Component {
       );
     }, 100);
 
+    setTimeout(async () => {
+      await this.props.sync();
+      await checkForUpdateNotification(
```

Comment:
It seems like this is run every time the app starts up. I don’t want to annoy people who decide to stay on older versions too much, would it be possible to hide the notification until the next Actual update is released when they click the X button and close the notification?

Reply:
That's a valid concern. Would storing this locally through the browser work for you or should this be saved on the server?

Reply:
If you search the code for `state.prefs.global` / `globalPrefs` you’ll see how several other components hook into the Redux store to persist global settings, and I think the version would be best persisted there.

Reply:
Thank you for the hint, I've changed it accordingly.